apps/meteor/client/views/admin/customEmoji/EditCustomEmoji.tsx

```diff
@@ -50,7 +51,7 @@ const EditCustomEmoji = ({ close, onChange, data, ...props }: EditCustomEmojiPro
 		}
 
 		if (data) {
-			return absoluteUrl(`/emoji-custom/${encodeURIComponent(data.name)}.${data.extension}`);
+			return absoluteUrl(`/emoji-custom/${encodeURIComponent(data.name)}.${data.extension}${data.etag ? `?etag=${data.etag}` : ''}`);
```

Comment:
I'm not sure this is right? Can you explain why eTag after the extension?

Reply:
The url of an emoji is basically depended on the emoji name. So even if the file changes. The url remained the same. Hence, the old emoji was loaded from the browser cache instead of the new file. 

The etag is generated for each new uploads. The is attached to url as query param to load the emoji image. 
This way the emoji will not load from the cache when the new file is upload, since the url will be changed.